DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  NetworkProtectionStatusView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+
+#if NETWORK_PROTECTION
```

Comment:
You may as well wrap the `import Swift` inside this `#if` so that the compiler doesn't even process it.

Reply:
I’ve got one more PR off the back of this with significant changes to that file, so I might just add this in that one, if that’s okay.